### ISSUE TITLE
save_device_attributes not sending data as json object

### ISSUE THREAD

**Issue by jernkuan** [thingsboard/thingsboard-python-rest-client#35]
Example code
```
      res = rest_client.save_device_attributes(
          body=json.dumps(claiming_json),
          device_id=device.id,
          scope='SERVER_SCOPE'
```

```
def save_device_attributes(self, body: str, device_id: DeviceId, scope: str):
```
requires a string type, but the final call to the rest endpoint, it uses json.dumps again

https://github.com/thingsboard/thingsboard-python-rest-client/blob/master/tb_rest_client/rest.py#L163
```
                if re.search('json', headers['Content-Type'], re.IGNORECASE):
                    request_body = '{}'
                    if body is not None:
                        request_body = json.dumps(body)
```

Reason:
HTTP response headers: HTTPHeaderDict({'vary': 'Origin, Access-Control-Request-Method, Access-Control-Request-Headers', 'x-content-type-options': 'nosniff', 'x-xss-protection': '1; mode=block', 'cache-control': 'no-cache, no-store, max-age=0, must-revalidate', 'pragma': 'no-cache', 'expires': '0', 'strict-transport-security': 'max-age=31536000 ; includeSubDomains', 'content-type': 'text/plain;charset=UTF-8', 'content-length': '28', 'date': 'Sat, 09 Oct 2021 05:35:40 GMT'})
HTTP response body: Request is not a JSON object

Probably this is affecting multiple API that are expecting body as string type.

**Comment by MielHostens** [comment]
Correct, with current SDK this is still an issue, changing the rest.py solves everything
